$DragonFly: src/gnu/lib/gcc40/libstdc++/Attic/ctype_base.h.patch,v 1.1 2005/06/05 22:43:21 corecode Exp $
Index: ctype_base.h
===================================================================
RCS file: /cvs/src/contrib/gcc-3.4/libstdc++-v3/config/os/bsd/netbsd/ctype_base.h,v
retrieving revision 1.3
diff -u -r1.3 ctype_base.h
--- config/os/bsd/netbsd/ctype_base.h	2 Mar 2005 00:56:21 -0000	1.3
+++ config/os/bsd/netbsd/ctype_base.h	7 Mar 2005 17:38:44 -0000
@@ -43,16 +43,16 @@
 
     // NB: Offsets into ctype<char>::_M_table force a particular size
     // on the mask type. Because of this, we don't use an enum.
-    typedef unsigned char      	mask;
+    typedef uint16_t      	mask;
     static const mask upper    	= _U;
     static const mask lower 	= _L;
-    static const mask alpha 	= _U | _L;
-    static const mask digit 	= _N;
-    static const mask xdigit 	= _N | _X;
+    static const mask alpha 	= _A;
+    static const mask digit 	= _D;
+    static const mask xdigit 	= _X;
     static const mask space 	= _S;
-    static const mask print 	= _P | _U | _L | _N | _B;
-    static const mask graph 	= _P | _U | _L | _N;
+    static const mask print 	= _R;
+    static const mask graph 	= _G;
     static const mask cntrl 	= _C;
     static const mask punct 	= _P;
-    static const mask alnum 	= _U | _L | _N;
+    static const mask alnum 	= _A | _D;
   };
